app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
- [ ] Good job so far adding an authentication callback method for _devise_, however, let's also ensure that CanCanCan is used for Post authorization in the _Post Controller_ so as to allow authorization of resources. We can achieve this by adding helper methods like the example below:

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```